lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1402,3 +1424,70 @@ def _get_deployable_vhd_path(
             wait_copy_blob(blob_client, vhd_path, log)
 
         return full_vhd_path
+
+    def _node_generate_min_capability(
+        self, req_cap: schema.NodeSpace, azure_cap: schema.NodeSpace
+    ) -> schema.NodeSpace:
+        assert azure_cap.features, "azure node has no disk features."
+        assert (
+            AzurePlatform._get_disk_feature_count(req_cap.features) <= 1
+        ), "Multiple disk feature found in node requirement."
+
+        if AzurePlatform._get_disk_feature_count(req_cap.features) == 0:
+            # If no disk type feature is specified, use least
+            # cost available disk from azure_cap
+            min_cost_disk = AzurePlatform._get_min_cost_disk(azure_cap.features)
+        else:
+            min_cost_disk = AzurePlatform._get_min_cost_disk(
+                req_cap.features  # type: ignore
+            )
+
+        # Remove mutually exclusive feature
+        azure_cap_copy = copy.deepcopy(azure_cap)
+        azure_cap_copy.features.update(  # type: ignore
+            copy.deepcopy(azure_cap.features.items)
+        )
+        for me_feature_group in features.MUTUALLY_EXCLUSIVE_FEATURES:
+            if min_cost_disk in me_feature_group:
```

Comment:
It's better to add an example and explanation to help understanding the purpose and logic.

Reply:
Sure! I will add examples to make it clear.

Reply:
I am thinking of adding following comment : 
```
# Remove features which are mutually exclusive feature to `min_cost_disk`
# Example :
# min_cost_disk = DiskStandardLRS
# (Initial) azure_cap_copy.features = [DiskStandardLRS, DiskPremiumLRS, DiskEphemeral]
# (Final) azure_cap_copy.features = [DiskStandardLRS]
```

Please feel free to suggest any edits.

Reply:
It's good to me. It can be more human readable, not in the code style.